deployments/helm/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 1.16.0
+appVersion: v0.9
```

Comment:
```suggestion
appVersion: v0.10.2
```
Or we can do this change when we update the stable release. WDYT?

Reply:
Then if ain't ever be updated, shouldn't we set it to something else so as it is not misleading or suggests it's unamanaged?